README.md

```diff
@@ -637,6 +637,7 @@ If there are no 'ready' tasks in the queue, returns nil.
 * `tube:bury(task_id)` - buries a task
 * `tube:kick(count)` - digs out `count` tasks
 * `tube:peek(task_id)` - return the task state by ID
+* `tube:tasks_by_state(task_state)` - return the iterator to tasks in a certain state
```

Comment:
Should [tube.grant](https://github.com/tarantool/queue/blob/de0a346f17e690aed19384577ed5157476f8fa3a/queue/abstract.lua#L267-L277) be updated to grant proper persissions to this new function?

Reply:
Nit: *Almost* all other public function names begin with a verb (except `stats`). Maybe we should stick to that convention and rename this function to `get_tasks_by_state()` or `get_by_state()`? Wdyt?

Reply:
No. It's about driver API, not about tube API.

Reply:
In this section of README read 'driver', don't read tube.

Reply:
> Should [tube.grant](https://github.com/tarantool/queue/blob/de0a346f17e690aed19384577ed5157476f8fa3a/queue/abstract.lua#L267-L277) be updated to grant proper persissions to this new function?

We call it only on start and don't provide a way to call it (from public API point of view), so I guess no.

Reply:
I would even name it just `:pairs(<...>)` with ability to provide different arguments. Now it'll be `{status = <...>}`.

Reply:
Let's keep the current method to fix the bug. If we'll agree how to better name it in a future — let's change then.